### ISSUE TITLE
2.6.5_dev watch app settings don't open

### ISSUE THREAD

**Issue by guydavies** [MilosKozak/AndroidAPS#2594]
I also updated my watch (TicwatchS2) to the image associated with the 2.6.5_dev build I've been using to test. I just noticed that if I try to open settings on the watch app it crashes and goes back to my list of apps. other functions appear to work OK and the watch faces continue to work based on previous settings. I reverted to the watch app matching 2.6.1 and it works as expected when you open settings. 

I don't have any logs to add from the watch. I can try to generate some if there's a way to do so.

**Comment by guydavies** [comment]
I've been looking at 2.6.5_dev on my main phone (open loop) and my watch. it seems like the watch app menu isn't really working at all. 

I've tried to set a Temp Target and that accepts the input but doesn't send it to the phone. After clicking on the tick, the app crashes and the next time I open the AAPSv2 watchface the traces are gone (as when the app first starts). 

Both settings and any sub-menu item of status crash the app going back to the list of apps in the launcher. Since there's no way to get to the Settings menu to confirm the version, it is probably the same one as the phone app (I deleted and reinstalled the app from the Play Store app). My phone is running 9995e7448.
